src/asciidoc/templates/document.html.erb

```diff
@@ -0,0 +1,273 @@
+      <ul>
+        <li><a href="/">About</a></li>
+        <li><a href="/download.html">Downloads</a></li>
+        <li>
+          <a href="#">Learn <i class="fa fa-chevron-circle-down" aria-hidden="true"></i></a>
+          <ul class="menu-drop-down">
+<!--            <li><a href="/getting-started/">Getting Started</a></li>-->
+<!--            <li><a href="#">Manual</a></li>-->
+<!--            <li><a href="#">Guides</a></li>-->
+            <li><a href="/legacy-docs.html">Old Docs</a></li>
+            <li><a href="https://cwiki.apache.org/confluence/display/LOGGING/Log4j">Wiki</a></li>
+            <li><a href="/articles.html">Articles</a></li>
+            <li><a href="/faq.html">FAQ</a></li>
+          </ul>
+        </li>
+        <li><a href="#">Develop <i class="fa fa-chevron-circle-down" aria-hidden="true"></i></a>
+          <ul class="menu-drop-down">
+            <li><a href="/release-notes.html">Release Notes</a></li>
+            <li><a href="/javadoc.html">Javadoc</a></li>
+            <li><a href="/runtime-dependencies.html">Runtime Dependencies</a></li>
+          </ul>
+        </li>
+        <li><a href="/support.html">Support</a></li>
+        <li><a href="#">Contribute <i class="fa fa-chevron-circle-down" aria-hidden="true"></i></a>
+          <ul class="menu-drop-down">
+            <li><a href="/guidelines.html">Guidelines</a></li>
+            <li><a href="/components.html">Components</a></li>
+            <li><a href="/javastyle.html">Style Guide</a></li>
+            <li><a href="https://analysis.apache.org/dashboard/index/org.apache.logging.log4j:log4j">Sonar</a></li>
+            <li><a href="https://github.com/apache/logging-log4j2">GitHub</a></li>
+            <li><a href="https://logging.apache.org/team-list.html">Team</a></li>
+          </ul>
+        </li>
+        <li><a href="/security.html">Security</a></li>
+      </ul>
+    </nav>
+
+    <nav class="secondary-menu" aria-label="ASF related navigation">
+      <ul>
+        <li>
+          <a href="#">Apache <i class="fa fa-chevron-circle-down" aria-hidden="true"></i></a>
+          <ul class="menu-drop-down">
+            <li><a target="_blank" href="https://www.apache.org/">Home</a></li>
+            <li><a target="_blank" href="https://www.apache.org/foundation/sponsorship.html">Sponsorship</a></li>
+            <li><a target="_blank" href="https://www.apache.org/licenses/">License</a></li>
+            <li><a target="_blank" href="https://www.apache.org/foundation/thanks.html">Thanks</a></li>
+            <li><a target="_blank" href="https://www.apache.org/events/current-event.html">Current Events</a></li>
+            <li><a target="_blank" href="https://www.apache.org/security/">Security</a></li>
+            <li><a target="_blank" href="https://privacy.apache.org/policies/privacy-policy-public.html">Privacy</a></li>
+          </ul>
+        </li>
+        <li><a href="https://logging.apache.org">Logging</a></li>
```

Comment:
Only saying _"Logging"_ is prone to confusion given the current context.
```suggestion
        <li><a href="https://logging.apache.org">Logging Services</a></li>
```

Reply:
This might break navigation if it is too long. We may need to come up with a better term. Added for now with 85b1161802726e9344e3e614f4b382d69cb08d77